dpnp/tests/third_party/cupy/linalg_tests/test_product.py

```diff
@@ -40,7 +40,7 @@
 )
 class TestDot(unittest.TestCase):
 
-    @testing.for_all_dtypes_combination(["dtype_a", "dtype_b"])
+    @testing.for_all_dtypes_combination(["dtype_a", "dtype_b"], no_int8=True)
```

Comment:
I believe it would be helpful to state why we skipped int8 and what caused the overflow for product functions.